### PR TITLE
[Docs] Update "Customization flashes "

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: docs/customization/flash.rst [hunk 1]
```diff
@@ -16,6 +16,12 @@ and many other places where you can customize the text content of the default fl
 How to customize a flash message?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. tip::
+
+    You can browse the full implementation of these examples on `this GitHub Pull Request.
+    <https://github.com/Sylius/Customizations/pull/18>`_
+
+
```
Comment (GSadee) [inline docs/customization/flash.rst]: Unnecessary blank line

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "docs-customizing-flashes"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: docs/customization/flash.rst [hunk 2]
```diff
@@ -36,4 +41,14 @@ the ``sylius.taxon.update`` key:
         taxon:
             update: This category has been successfully edited.
 
+Before
+
+.. image:: ../_images/flash_before_customization.png
+    :align: center
+
+after
```
Comment (Zales0123) [inline docs/customization/flash.rst]: After

**Comment by GSadee** [comment]
Thank you, Adam! :1st_place_medal:

**Pull request merged**